src/domain/synthetics/tokens/use24Volume.ts

```diff
@@ -0,0 +1,60 @@
+import { gql } from "@apollo/client";
+import { selectChainId } from "context/SyntheticsStateContext/selectors/globalSelectors";
+import { selectTradeboxMarketInfo } from "context/SyntheticsStateContext/selectors/tradeboxSelectors";
+import { useSelector } from "context/SyntheticsStateContext/utils";
+import { getSyntheticsGraphClient } from "lib/subgraph";
+import { useCallback, useEffect, useState } from "react";
+
+export function use24hVolume() {
+  const chainId = useSelector(selectChainId);
+  const marketInfo = useSelector(selectTradeboxMarketInfo);
+  const client = getSyntheticsGraphClient(chainId);
+
+  const [value, setValue] = useState<bigint>(0n);
+
+  const fetch = useCallback(
+    async function call() {
+      if (!client || !marketInfo?.indexTokenAddress) {
+        return;
+      }
+
+      const LAST_DAY_UNIX_TIMESTAMP = Math.floor(Date.now() / 1000) - 24 * 60 * 60;
+      const utcOffsetSecs = new Date().getTimezoneOffset() * 60;
+      const timestamp = LAST_DAY_UNIX_TIMESTAMP + utcOffsetSecs;
```

Comment:
1 we have a TIMEZONE_OFFSET_SEC variable
2 why this logic with offsetting is needed, does the backend store local timestamps ?

Reply:
1. Thanks!
2. Backend in UTC, this code will show 24h volume in UTC timezone (that would be weird if users in different tzs will see different trading volume), but will doublecheck with council of elders

Reply:
sounds ok